osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -560,5 +581,15 @@ private string getParentMessage()
                 return parentComment.HasMessage ? parentComment.Message : parentComment.IsDeleted ? "deleted" : string.Empty;
             }
         }
+
+        public Popover GetPopover() => new ReportCommentPopover(this);
+
+        private class ReportToast : Toast
+        {
+            public ReportToast()
+                : base(UserInterfaceStrings.GeneralHeader, UsersStrings.ReportThanks, "")
+            {
+            }
+        }
```

Comment:
Looking at this again, maybe text placed in place of the "report" button would feel better than a toast:

https://user-images.githubusercontent.com/22781491/196823678-ceec56b6-5a01-4b3c-bb0c-a77a47835994.mp4

@peppy thoughts?

Reply:
The text proposal has already been pushed in this PR, but I'll wait for further opinions before merge.